unitTests/utils/DomTest.ts

```diff
@@ -615,6 +647,103 @@ export function DomTests() {
         Simulate.removeJQuery();
       });
 
+      describe('with native set to true', () => {
+        beforeEach(() => {
+          Dom.native(true);
+        });
+
+        afterEach(() => {
+          Dom.native(false);
+        });
+
+        it('using "on" should work properly', () => {
```

Comment:
I would prefer to break this test into 3, each with a good description of what the test is doing. e.g.

Test 1: `when using #on to bind a click event, when triggering a click, it calls the click handler`
Test 2: `when using #on to bind a custom event, when triggering the custom event with a payload, it calls the handler with the payload`.
Test 3: `when using #on to bind three different events having the same handler, when triggering the three events, it calls the handler three times`.

Reply:
I see you are use to the ruby method referencing convention : p.